### PR TITLE
Survey

### PR DESCRIPTION
**Pull request by mian-saad**

### CODE REVIEW

File: survey/survey.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    margin-top: 0px;
+}
+
+.form-wraper{
+    background-color: rgb(251, 252, 245);
+    text-align: center;
+    max-width: 800px;
+    margin: auto;
+    padding: 10px;
+    box-shadow: 0px 0px 8px 2px gray;
```
Comment (ahsanz024) [inline survey/survey.css]: just 0 instead of 0px

File: survey/survey.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.labls{
+    display: inline-block;
+    text-align: right;
+    width: 40%;
+    padding: 5px;
+}
+
+.labls1{
```
Comment (ahsanz024) [inline survey/survey.css]: should be meaningful names

File: survey/survey.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    width: 48%;
+    padding: 5px;
+}
+
+textarea{
+    height: 40px;
+    resize: vertical;
+}
+
+button{
```
Comment (ahsanz024) [inline survey/survey.css]: use class

File: survey/survey.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                                <li><label><input name="name" value="Three" type="radio">Three</label></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="rw">
+                        <div class="labls labls1">
+                            <label>Things that should be improved in the future (Check all that apply):</label>
+                        </div>
+                        <div class="ins">
+                            <ul class="style">
```
Comment (ahsanz024) [inline survey/survey.html]: could be `list-style` instead of `style`

File: survey/survey.css (hunk 1 above)
Comment (mian-saad) [reply survey/survey.css]: 0 does not work

File: survey/survey.css (hunk 3 above)
Comment (mian-saad) [reply survey/survey.css]: done

File: survey/survey.html (hunk 4 above)
Comment (mian-saad) [reply survey/survey.html]: done

File: survey/survey.css (hunk 2 above)
Comment (mian-saad) [reply survey/survey.css]: done

File: survey/survey.css (hunk 1 above)
Comment (ahsanz024) [reply survey/survey.css]: `box-shadow: 0 0 8px 2px gray;`
doesn't work?
its not something you have to do, its just a convention

File: survey/survey.css [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.ins{
+    display: inline-block;
+    text-align: left;
+    width: 48%;
+    padding: 5px;
+}
+
+textarea{
```
Comment (ahsanz024) [inline survey/survey.css]: not using classes again

File: survey/survey.css (hunk 2 above)
Comment (ahsanz024) [reply survey/survey.css]: still `labls1`?

File: survey/survey.css (hunk 1 above)
Comment (mian-saad) [reply survey/survey.css]: ahso
I thought  
`box-shadow: 0 0 8 2 gray;`
`box-shadow: 0 0 8px 2px gray;` works. Thanks

File: survey/survey.css (hunk 5 above)
Comment (mian-saad) [reply survey/survey.css]: done

**Review by ahsanz024 — APPROVED** [review]
Nice!!

**Pull request merged**